test/sequential/test-watch-mode.mjs

```diff
@@ -74,11 +80,31 @@ function assertRestartedCorrectly({ stdout, messages: { inner, completed, restar
   assert.deepStrictEqual(lines.slice(-end.length), end);
 }
 
+async function failWriteSucceed({ file, watchedFile }) {
+  let stdout = '';
+  const notFound = deferred();
+  const completed = deferred();
+  const child = spawn(execPath, ['--watch', '--no-warnings', file], { encoding: 'utf8' });
+
+  child.stdout.on('data', (data) => {
+    console.log(data.toString());
+    stdout += data;
```

Comment:
Probably neater to `toString()` explicitly here if it's done in the console.log above?